packages/node_modules/@webex/widget-space/src/enhancers/setup.js

```diff
@@ -41,6 +42,7 @@ export function setup(props, prevProps = {}) {
   ) {
     // Check if we need to reload space
     if (widgetStatus.shouldReloadWidget && prevProps && !prevProps.widgetStatus.shouldReloadWidget) {
+      props.resetConversation();
```

Comment:
Moved from `message-widget` to here for a single-call status update on the request: `isFetching`